src/config/index.ts

```diff
@@ -0,0 +1,4 @@
+// TODO(fjv): Temporäre Lösung für eine Zentrale Konfigurationsdatei. Später in einer .env (dotenv) Datei?
```

Comment:
Ja kann man so machen und wird oft so gemacht :)

Reply:
zu lösen in #87